style.css

```diff
@@ -549,6 +550,69 @@ body > footer {
 	}
 }
 
+@media (max-width: 1000px){
+	body {
+		grid-template:
+			'header header' auto
+			'curve preview' auto
+			'curve library' auto / 300px 1fr;
+	}
+
+	header {
+		text-align: center;
+	}
+
+	header > h1 {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: center;
+		align-items: center;
+	}
+}
+
+@media (max-width: 850px){
+	body {
+		grid-template:
+			'footer' auto
+			'header' auto
+			'curve' auto
+			'preview' auto
+			'library' auto / auto;
+		padding: 0 5% 1em;
+	}
+
+	body > footer {
+		padding: .5em 0;
+		position: static;
+		max-width: 100%;
+		transform: none;
+		white-space: normal;
+	}
+
+	h1 > a {
+		font-size: 0.8em;
+		width: 100%;
+	}
+
+	#copybuttons {
+		margin-left: auto;
+	}
+}
+
+@media (max-width: 600px){
+	h1 > a {
+		font-size: 0.7em;
+		letter-spacing: -.05ch;
+	}
+}
+
+@media (max-width: 450px){
+	h1 > a {
+		font-size: 0.6em;
+		letter-spacing: -.05ch;
```

Comment:
We don't need `letter-spacing` here, since 450px is also < 600px so the MQ above applies as well.